android/sdl_android/src/main/java/com/smartdevicelink/managers/SdlManager.java

```diff
@@ -1032,6 +1032,15 @@ public SdlManager build() {
 				sdlManager.isMediaApp = false;
 			}
 
+			if(TransportType.MULTIPLEX.equals(sdlManager.transport.getTransportType())){
```

Comment:
potential NPE

Reply:
Moving the logic to start() for the SdlManager instead of the build function. Side note, should the build function throw an IllegalArgumentException if no transport is set at the time of the build called? I don't believe we ever create one if it's not supplied.

Reply:
I agree it should throw an `IllegalArgumentException` and that setting the transport be `@NonNull`